vhd/packer/install-dependencies.sh

```diff
@@ -285,6 +285,8 @@ for AZURE_CNI_NETWORKMONITOR_VERSION in ${AZURE_CNI_NETWORKMONITOR_VERSIONS}; do
 done
 
 AZURE_NPM_VERSIONS="
+1.1.1
+1.1.0
```

Comment:
Any reason why we're adding 1.1.0 to the VHD?

Reply:
It looks like we missed it last time

Reply:
Right, but is there a reason to keep > 1 version on the VHD? Since AKS Engine doesn't offer a convenient way to roll back and addon, I think we should just keep 1 version here.

Reply:
sure, I will remove it since it isn't required